src/mds/MDSMap.h

```diff
@@ -234,6 +234,15 @@ class MDSMap {
   bool allows_standby_replay() const { return test_flag(CEPH_MDSMAP_ALLOW_STANDBY_REPLAY); }
   bool was_standby_replay_ever_allowed() const { return ever_allowed_features & CEPH_MDSMAP_ALLOW_STANDBY_REPLAY; }
 
+  void set_reject_client_session() {
+    set_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION);
+    ever_allowed_features |= CEPH_MDSMAP_REJECT_CLIENT_SESSION;
+    explicitly_allowed_features |= CEPH_MDSMAP_REJECT_CLIENT_SESSION;
+  }
+  bool get_reject_client_session() const { return test_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION); }
+  void clear_reject_client_session() { clear_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION); }
+  bool was_ever_client_rejected_session() const { return ever_allowed_features & CEPH_MDSMAP_REJECT_CLIENT_SESSION;}
```

Comment:
Ah so all of them can be removed but I opine that we keep `get_reject_client_session()` caue it's being used a lot of times outside `MDSMap.cc` or `FSCommands.cc` and doing `mdsmap.test_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION)` in `if` conditions would look too ugly.

Reply:
in `void MDSMap::print_flags`, it would be like `if (test_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION)` while others are like `if (allows_standby_replay())`. I hope that non-uniformity is okay?

Reply:
it's okay I think.

Reply:
okay, cool